docs-gb/examples/experiment-versions.md

```diff
@@ -41,7 +41,7 @@ seldon model infer iris -i 50 \
   '{"inputs": [{"name": "predict", "shape": [1, 4], "datatype": "FP32", "data": [[1, 2, 3, 4]]}]}'
 ```
 
-```
+```bash
```

Comment:
Those are outputs rather than bash (everything that starts with Success)

Reply:
fixed.